### PR TITLE
Csstest

### PR DESCRIPTION
**Pull request by mbosselmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# Project Decription
+# Project Description
```
Comment (macfrei) [inline README.md:1]: Nice! 👍

File: index.html [hunk 2]
```diff
@@ -6,25 +6,29 @@
...
+            <a class="nav-links" href="#contact">Contact</a>
+            <a class="nav-links" href="#bio">About me</a>
+            <a class="nav-links" href="#cv">CV</a>
         </nav>
     </header>
     <main>
         <section id="short-intro">
             <figure>
+                <p> Welcome to my wonderful page. My Name is Mareike and I'm from Germany, currently living in
+                    Schleswig-Holstein.</p>
```
Comment (macfrei) [inline index.html:28]: Maybe check out `figcaption` for the `figure` element.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    flex-wrap: wrap;
+    background-color: #017A8B;
+    box-shadow: 2px 0px 5px lightgrey;
+}
+
+
+body {
+    font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    margin: 0;
+}
```
Comment (macfrei) [inline styles.css:22]: The `body` contains all your other HTML and usually the way we structure the CSS is to go from the Elements which contain other elements (broad elements like `html`, `*`, `body`) and then go to other. Long story short, I'd declare the styles for the `header` beneath the styles for the `body`.

File: index.html (hunk 2 above)
Comment (mbosselmann) [reply index.html:28]: Thank you! Changed it.

File: styles.css (hunk 3 above)
Comment (mbosselmann) [reply styles.css:22]: Ok! Switched body and header. You'll see the the changes in my next pull request (i think nr. 11).

Comment (macfrei) [reply styles.css:22]: Nice, I am looking forward to it!

**Pull request merged**